views/managing-officer-corporate.html

```diff
@@ -38,6 +38,28 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">Tell us about the corporate managi
       {% set dateText = "When did it become a managing officer for the overseas entity?" %}
       {% include "includes/inputs/date/start-date-input.html" %}
 
+      <h2 class="govuk-heading-l">Who can we contact about this managing officer?</h2>
+
+      {{ govukInput({
+        label: {
+          text: "What is their full name?",
+          classes: "govuk-label--m",
+          isPageHeading: false
+        },
+        id: "contact-full-name",
```

Comment:
could you also change the hyphens to underscores for the contact fields in this template